specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/PartnerListTransfers.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "api-version": "2023-04-01",
+    "billingAccountName": "{billingAccountName}",
+    "customerName": "{customerName}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/BillingAccounts/{billingAccountName}/customers/{customerName}/transfers/{transferName}",
```

Comment:
I know that there have been internal discussions as a team about scoping customer paths to /billingAccount/billingProfile/customer. Please confirm this path and all other customer paths with Ram/Amber.